src/BusyIndicator/BusyIndicator.js

```diff
@@ -0,0 +1,63 @@
+import { BUSY_INDICATOR_SIZES } from '../utils/constants';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+
+const BusyIndicator = React.forwardRef(({
+    children,
+    className,
+    show,
+    size,
+    disableStyles,
+    ...props
+}, ref) => {
+
+    useEffect(() => {
+        if (!disableStyles) {
+            require('fundamental-styles/dist/icon.css');
+            require('fundamental-styles/dist/busy-indicator.css');
+        }
+    }, []);
+
+    const busyIndicatorClasses = classnames(
+        {
+            [`fd-busy-indicator--${size}`]: !!size
```

Comment:
I am looking at https://fundamental-styles.netlify.com/components/busy-indicator.html
and I notice the small busy indicator class doesn't have a `--s` on it.  If you toggle between `fd-busy-indicator` and `fd-busy-indicator--s` there are style differences.

Maybe that linked doc is out of date, but I think there should be a special case for `size === 's'` here

Reply:
Noted. Let me incorporate above suggestion.

Reply:
Added above suggested changes.